Model/Ui/ConfigProvider.php

```diff
@@ -61,6 +67,8 @@ public function __construct(
         $this->gatewayConfig = $gatewayConfig;
         $this->storeId = $storeManager->getStore()->getId();
         $this->encoder = $encoder;
+        $this->isInternational = $storeManager->getStore()->getCurrentCurrencyCode() !== self::BRAZILIAN_CURRENCY;
```

Comment:
I think there was a little communication error.
The new message should be displayed for all Brazilian payments (and only when the country is `br`), regardless of the currency code.

This happens because this Magento Plugin is used for Cross Border only and all payments will be subject to foreign exchange transactions.

> Obs: We do not need to worry about Local payments, as there is a specific Magento plugin for that.